api/doc/openapi.yaml

```diff
@@ -143,13 +109,11 @@ paths:
     get:
       summary: Get account resources by ledger version
       operationId: get_account_resources_by_ledger_version
-      description: >-
+      description: |
```

Comment:
changed all `>-` to `|`, because `|` preserves linebreaks, thus you see I deleted a lot of blank lines in the following text